src/fw/drivers/ambient_light_opt3001.c

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright 2024 Google LLC
```

Comment:
ooops working for google?

Reply:
oopsies.  I can't remember if I wrote the contract for this to have open source IP belong to Accelerated Tech or to Core Devices, so I just assigned it to Core Devices this time